### ISSUE TITLE
page not found in `benchmark/users/README.md`

### ISSUE THREAD

**Issue by btrb** [opesci/devito#910]
page `https://perf.readthedocs.io/en/latest/system.html` not found in lines 68/69 of file `benchmark/users/README.md`:

```
Some more information about pinning is available
[here](https://perf.readthedocs.io/en/latest/system.html).
```

**Comment by georgebisbas** [comment]
Maybe we can use that:
 - https://buildmedia.readthedocs.org/media/pdf/perf/latest/perf.pdf ?

**Comment by georgebisbas** [comment]
Can be closed after #923 is merged

**Comment by FabioLuporini** [comment]
closed
